examples/v1beta1/pipelineruns/alpha/pipeline-object-results.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-object-results
+spec:
+  pipelineSpec:
+    tasks:
+      - name: task1
+        taskSpec:
+          results:
+            - name: object-results
+              type: object
+              description: The object results
+              properties:
+                foo: {
```

Comment:
are the curly braces necessary since type: string is already indented

Reply:
yes they are not. I will remove these